### PR TITLE
Move Crypto to src

### PR DESCRIPTION
**Pull request by zeroadam**
Move crypto to Friendica\Util namespace.

Related to #3878

### CODE REVIEW

File: src/Util/Crypto.php [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+		$der = self::pkcs1Encode($m, $e);
+		$key = self::DerToRsa($der);
+		return $key;
+	}
+
+	/**
+	 * @param string $pubkey public key
+	 * @return string
+	 */
+	public static function salmonKey($pubkey)
```
Comment (MrPetovan) [inline src/Util/Crypto.php]: Shouldn't this be moved to `Protocol\Salmon`?

**Comment by zeroadam** [comment]
I was thinking about this as I was going through, and thought you might
comment @MrPetovan!

File: src/Protocol/Salmon.php [hunk 2]
```diff
@@ -201,4 +200,14 @@ public static function slapper($owner, $url, $slap)
 
 		return (($return_code >= 200) && ($return_code < 300)) ? 0 : 1;
 	}
+
+	/**
+	 * @param string $pubkey public key
+	 * @return string
+	 */
+	public static function salmonKey($pubkey)
```
Comment (MrPetovan) [inline src/Protocol/Salmon.php]: I wanted to ask you to rename this function with its actual function, but I'm actually unable to figure out what it is doing exactly, so I guess we'll leave it at that 😂

Comment (annando) [reply src/Protocol/Salmon.php]: It encodes the public key of the user that is needed for Salmon notifications.

Comment (MrPetovan) [reply src/Protocol/Salmon.php]: So `Salmon::encodePubKey`?

Comment (annando) [reply src/Protocol/Salmon.php]: Doesn't sound worse than the existing name.

Comment (MrPetovan) [reply src/Protocol/Salmon.php]: I appreciate your enthusiastic support!

**Pull request merged**